### PR TITLE
feature highlights

### PR DESCRIPTION
**Pull request by juliapradel**
Bitte Korrektur lesen und unten ergänzen

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -10,6 +10,40 @@ Description
 
 Feature Highlights
 
+* **Create articles dynamically with paragraphs**: Using paragraphs, you can add text, pictures, videos, Instagram or Twitter cards to your article – and change the order by dragging and dropping the content wherever you like it. Thanks to a preview mode, you get a good idea of how your article will look like in the end.
+* **Working in distributed teams**: Thunder is a web-based system, so you can add and edit articles from everywhere. All you need is a computer and access to the Internet. Every modification to an article is saved in revisions. So, you can easily understand who did changes to your article. Plus, you can share a link to an unpublished article with external co-workers and give them access for a limited time. 
+* **User management**: In Thunder, you can define lots of different roles for every need—and allocate various rights.
+* **Liveblog**: This module helps to cover events as they happen, like sport events or elections.
```
Comment (dbosen) [inline README.md]: Its not yet integrated, and will not be released before drupal 8.3 is released in April. So maybe we should not have it in the featurelist as of now.

File: README.md [hunk 2]
```diff
@@ -10,6 +10,40 @@ Description
...
+* **Single Sign-On solution**: The Harbourmaster is a multi-site, multi-device Single Sign-On solution, developed for the media industry by our industry partner Valiton. The Harbourmaster contains a Policy Server, User Directory, Authentication Gateway, and User Manger.
+* **Facebook Instant Articles**: With Thunder, editors may publish their content easily as instant articles on Facebook. This reduces loading times on the mobile web.
+* **Analytics**: Interfaces to Google Analytics and the German audience measurement organization IVW.
+* **Ad-Integration**: We integrated Google AdSense. This enables you to embed advertisements into your website and earn money without developing your own solution.
+* **Demo Content and Guided Tour**: We added demo content and a guided tour to make it easier for newcomers to understand how Thunder works. Step by step, we explain how to add an article, how to edit an article, or how to add and sort channels. 
+* **Arrange your content**: Use categories and tags to sort your articles. Plus, you can easily create lists of content, such as the most read articles, or a package about a specific topic.
+
+NOCH OFFEN
+* Feeds: ????
```
Comment (dbosen) [inline README.md]: leave out, not an important feature

File: README.md [hunk 3]
```diff
@@ -10,6 +10,40 @@ Description
...
+* Multilingual: ???
```
Comment (dbosen) [inline README.md]: That means, that you can have a web page in multiple languages. E.G. a german and an english version.

File: README.md [hunk 4]
```diff
@@ -10,6 +10,40 @@ Description
...
+* Structure articles
+  * WYSIWYG-Editor
+  * verschiedene Ansichten (Teaser)
+* Technical details
+  * Scalability
+    * Caching system
+  * Migration support
+  * Big community
+    * Support system
+  * Rest API/ decoupled frontend
```
Comment (dbosen) [inline README.md]: I would leave this out of the core features.

File: README.md [hunk 5]
```diff
@@ -10,6 +10,40 @@ Description
...
+
+NOCH OFFEN
+* Feeds: ????
+* Multilingual: ???
+* Structure articles
+  * WYSIWYG-Editor
+  * verschiedene Ansichten (Teaser)
+* Technical details
+  * Scalability
+    * Caching system
```
Comment (dbosen) [inline README.md]: Caching system can be left out

File: README.md [hunk 6]
```diff
@@ -10,6 +10,40 @@ Description
...
+* **Ad-Integration**: We integrated Google AdSense. This enables you to embed advertisements into your website and earn money without developing your own solution.
+* **Demo Content and Guided Tour**: We added demo content and a guided tour to make it easier for newcomers to understand how Thunder works. Step by step, we explain how to add an article, how to edit an article, or how to add and sort channels. 
+* **Arrange your content**: Use categories and tags to sort your articles. Plus, you can easily create lists of content, such as the most read articles, or a package about a specific topic.
+
+NOCH OFFEN
+* Feeds: ????
+* Multilingual: ???
+* Structure articles
+  * WYSIWYG-Editor
+  * verschiedene Ansichten (Teaser)
```
Comment (dbosen) [inline README.md]: Teaser system is out of scope for feature highlights

File: README.md [hunk 7]
```diff
@@ -10,6 +10,40 @@ Description
...
+* Technical details
+  * Scalability
+    * Caching system
+  * Migration support
```
Comment (dbosen) [inline README.md]: Also out of scope

File: README.md [hunk 8]
```diff
@@ -10,6 +10,40 @@ Description
...
+  * Big community
+    * Support system
+  * Rest API/ decoupled frontend
+  * Themeable
+  * Extendable with 2500+ modules
+  * Tested with 10.000+ automated tests
```
Comment (dbosen) [inline README.md]: Tests are out of scope

File: README.md (hunk 3 above)
Comment (juliapradel) [reply README.md]: Buttons; automatisch auf deutsche Seite kommen; Texte verlinkt

**Pull request merged**